webrtc-stats.html

```diff
@@ -503,6 +511,96 @@
                   [[!RFC7509]] section 3.1 and Appendix A.b.
                   </p>
                 </dd>
+                <dt>
+                  <dfn><code>burstPacketsLost</code></dfn> of type <span class=
+                  "idlMemberType"><a>unsigned long</a></span>
+                </dt>
+                <dd>
+                  <p>
+                  The cumulative number of RTP packets lost during loss
+                  bursts, Appendix A (c) of [[!RFC6958]].
+                  </p>
+                </dd>
+                <dt>
+                  <dfn><code>burstPacketsDiscarded</code></dfn> of type <span class=
+                  "idlMemberType"><a>unsigned long</a></span>
+                </dt>
+                <dd>
+                  <p>
+                  The cumulative number of RTP packets discarded during
+                  discard bursts, Appendix A (b) of [[!RFC7003]].
+                  </p>
+                </dd>
+                <dt>
+                  <dfn><code>burstLossCount</code></dfn> of type <span class=
+                  "idlMemberType"><a>unsigned long</a></span>
+                </dt>
+                <dd>
+                  <p>
+                  The cumulative number of bursts of lost RTP packets,
+                  Appendix A (e) of [[!RFC6958]].
+                  </p>
+                  <p>[[!RFC3611]] recommends a Gmin (threshold) value of
+                  16 for classifying packet loss or discard burst.
```

Comment:
should this be "for classifying a sequence of packet losses or discards as a burst"?